package/network/services/ppp/Makefile

```diff
@@ -9,20 +9,21 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=ppp
-PKG_RELEASE:=4
+PKG_RELEASE:=5
```

Comment:
```suggestion
PKG_RELEASE:=1
```

Reply:
could you please explain more what strategy of assigning of local version? once merged with upstream then start from 1? Why do we need local  version at all?

Reply:
done, but please answer anyway.

Reply:
ok, i figure it out...

Reply:
The local or package version indicates this is the N-th version of the upstream sources. It should be incremented when we add changes like patches, build flags, or similar on top.

E.g. upstream version 2.4.9-1 is the first build of upstream 2.4.9
We add a patch to fix something or backport a feature, but still operate on the 2.4.9 upstream sources, then we package 2.4.9-2 and so on...

Bumping the upstream sources we should then reset the package version, s.t. we will produce let’s say 2.5.0-1, our first package based on 2.5.0 upstream. Does not matter if we still have custom patches on top or what the actual upstream changes are, it’s still the first package version that will be built.